code/modules/clothing/spacesuits/rig/suits/civilian.dm

```diff
@@ -17,3 +17,33 @@
 		/obj/item/rig_module/healthbar,
 		/obj/item/rig_module/storage/light
 		)
+
+/obj/item/weapon/rig/emergency
+	name = "Emergency RIG"
+	desc = "A light, emergency rig for use by non-qualified personnel in the case of emergency decompression."
+	icon_state = "eva_suit"
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 50)
+	online_slowdown = 1
```

Comment:
this is a bit too lightweight. being easy to move in is an advantage of some of the fancier rigs like the Advanced rig, it shouldn't be given out easily

Reply:
It has no stats practically and has the same online and offline slowdown as the firesuit.

It is a glass cannon.